.circleci/config.yml

```diff
@@ -35,13 +59,26 @@ jobs:
           paths:
             - dist
 
+  test_python: # Python tests are separated from YAML tests, because the latter take longer. This would allow Python tests to use any other available container and yield it, once finished, to another job instead of counting only on the two containers assigned to the YAML tests.
```

Comment:
```suggestion
  test_python: # Python tests are separated from YAML tests, because the latter take longer. This allows Python tests to use any other available container and yield it, once finished, to another job instead of relying only on the containers assigned to the YAML tests.
```